schemas/test/en/test_placeholder_option_label_from_value.json

```diff
@@ -0,0 +1,266 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Test option label from value",
+    "theme": "default",
+    "description": "A questionnaire to demo the piping of Checkbox/Radio answer options",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
+    "sections": [
+        {
+            "id": "checkbox-section",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "mandatory-checkbox",
+                            "question": {
+                                "answers": [
```

Comment:
We should add some guidance on this question page on how to use this schema. We do this with a couple of schemas. It will make it easier to know how to test. So something simple guidance to state, scenario one, choose 1 option which will be piped into the question title, and scenario two, pick more than 1, and the first selected answer will be piped into a radio etc.